src/main/resources/META-INF/plugin.xml

```diff
@@ -30,7 +30,7 @@
     </change-notes>
 
     <!-- please see http://www.jetbrains.org/intellij/sdk/docs/basics/getting_started/build_number_ranges.html for description -->
-    <idea-version since-build="171.0" until-build="172.0"/>
+    <idea-version since-build="171.3780" until-build="172.0"/>
```

Comment:
Is this the version when intellij introduced the new scala plugin or just your _current_ version? (however, I'd be fine with both, it's still eap anyway)